src/ripple/app/ledger/InboundLedger.cpp

```diff
@@ -1120,7 +1120,7 @@ int InboundLedger::processData (std::shared_ptr<Peer> peer,
     if ((packet.type () == protocol::liTX_NODE) || (
         packet.type () == protocol::liAS_NODE))
     {
-        std::list<SHAMapNodeID> nodeIDs;
+        std::vector<SHAMapNodeID> nodeIDs;
         std::list< Blob > nodeData;
```

Comment:
It didn't come onto my radar because we don't call `size()` with it.  However you're correct that this is another example.  I'll change this to `vector`.  Good catch!